src/wp-includes/class-wp-phpmailer.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * PHPMailer extensions for WordPress
+ *
+ * @package WordPress
+ * @since 6.7.1
+ */
+
+/**
+ * PHPMailer child class
+ *
+ * Overrides the internationalization method in order to use WordPress' instead.
+ *
+ * @since 6.7.1
+ */
+class WP_PHPMailer extends PHPMailer\PHPMailer\PHPMailer {
+	/**
+	 * Public constructor
+	 *
+	 * @param bool $exceptions Whether to throw exceptions for errors.
+	 */
+	public function __construct( $exceptions = false ) {
+		parent::__construct( $exceptions );
+		$this->SetLanguage();
+	}
+
+	/**
+	 * Defines the error messages using WordPress' internationalization method.
+	 *
+	 * @return bool Always returns true in order to mimic PHPMailer's setLanguage() behavior
+	 */
+	public function SetLanguage( $langcode = 'en', $lang_path = '' ) {
+		$error_strings  = array(
+			'authenticate'         => __( 'SMTP Error: Could not authenticate.' ),
+			'buggy_php'            => __(
+				'Your version of PHP is affected by a bug that may result in corrupted messages. To fix it, switch to sending using SMTP, disable the mail.add_x_header option in your php.ini, switch to MacOS or Linux, or upgrade your PHP to version 7.0.17+ or 7.1.3+.'
+			),
```

Comment:
The last part is weird, since WordPress requires PHP 7.2+ anyway, so maybe a good opportunity to rewrite that.

```suggestion
			'buggy_php'            => sprintf(
				/* translators: 1: mail.add_x_header. 2: php.ini */
				__(
					'Your version of PHP is affected by a bug that may result in corrupted messages. To fix it, switch to sending using SMTP, disable the %1$s option in your %2$s, or switch to MacOS or Linux, or upgrade your PHP version.'
				),
				'mail.add_x_header',
				'php.ini',
			),
```